docs/io/fits/usage/headers.rst

```diff
@@ -336,7 +336,7 @@ for keywords, a proposal was made in:
     https://heasarc.gsfc.nasa.gov/docs/heasarc/ofwg/docs/ofwg_recomm/r13.html
 
 by using the CONTINUE keyword after the regular 80 column containing the
-keyword. ``astropy`` does support this convention, even though it is not a FITS
+keyword. ``astropy`` does support this convention, which is a part of latest FITS 
```

Comment:
"latest" is a moving target. Even though the FITS standard changes very rarely I would feel better if we write the version explicitly "which is a part of the FITS standard since version 4" (it is 4, right?).

Also the trailing whitespace at the end of the line can be removed.

Reply:
Yes, it's written 4.0 on the website. I have made the change you suggested. Thank's @MSeifert04 for your remarks.